api/task_manager.cc

```diff
@@ -176,7 +176,9 @@ void set_task_manager(http_context& ctx, routes& r) {
         auto task = co_await tasks::task_manager::invoke_on_task(ctx.tm, id, std::function([] (tasks::task_manager::task_ptr task) {
             return task->done().then_wrapped([task] (auto f) {
                 task->unregister_task();
-                f.get();
+                // done() is called only because we want the task to be complete before getting its status.
+                // The future should be ignored here as the result does not matter.
+                f.ignore_ready_future();
```

Comment:
Won't change this in a backport, but consider changing the future to return a status instead of something that needs to be ignored and then someone else has to check the status.

Reply:
but: how is it cherry-picked from d6da3db298e3ca6aabe849f7e0667e6ccc154482? it's not in master.

Reply:
There are other layers that need the current behavior of `done()`

Reply:
> but: how is it cherry-picked from [d6da3db](https://github.com/scylladb/scylladb/commit/d6da3db298e3ca6aabe849f7e0667e6ccc154482)? it's not in master.

Oh, sorry, I forgot that single commits are cherry-picked to master and took the original commit, will change